homeassistant/components/template/fan.py

```diff
@@ -231,7 +314,7 @@ async def async_turn_on(
 
         if preset_mode is not None:
             await self.async_set_preset_mode(preset_mode)
-        elif percentage is not None:
+        if percentage is not None:
```

Comment:
Why is this changed from `elif` to `if`?

Reply:
Ah yes, this was changed because if you supply both preset_mode and percentage (which is valid for fans) in the turn_on service call, it wouldn't run both actions, it would just run preset_mode.